src/main/java/teammates/ui/controller/PageData.java

```diff
@@ -1001,6 +1002,10 @@ public boolean isCourseArchived(String courseId, String googleId) {
         return Logic.isCourseArchived(courseId, googleId);
     }
     
+    public boolean isCourseArchived(CourseAttributes course, InstructorAttributes instructor) {
+        return Logic.isCourseArchived(course, instructor);
+    }
```

Comment:
Right, deleting it from PageData has no repercussions. At least not in Java side. Need to test.